dev-tools/reconfigurator-cli/tests/output/cmds-set-remove-mupdate-override-stdout

```diff
@@ -301,6 +301,7 @@ internal DNS:
         AAAA fd00:1122:3344:101::1
     name: 9a867dc9-d505-427f-9eff-cdb1d4d9bd73.sled          (records: 1)
         AAAA fd00:1122:3344:106::1
+    name: @                                                  (records: 0)
```

Comment:
hm. that looks like a bug. there are no DNS zones here, so there are correctly no records, but there shouldn't be even an empty list if there are no records for the name.

Reply:
[it was a bug](https://github.com/oxidecomputer/omicron/pull/8047/commits/3821a1f8eeb3875c1cbbecd424e6593dd15ebf2d)